bin/qds.py

```diff
@@ -238,6 +249,7 @@ def _create_cluster_info(arguments, api_version):
                                       aws_availability_zone=arguments.aws_availability_zone,
                                       vpc_id=arguments.vpc_id,
                                       subnet_id=arguments.subnet_id,
+                                      master_elastic_ip=arguments.master_elastic_ip,
```

Comment:
Apart from the testing of include_query_properties, I am guessing everything else is already taken care by a different PR. Can you merge the latest branch?